### ISSUE TITLE
Faucet not creating account

### ISSUE THREAD

**Issue by 0xPrimata** [movemntdev/faucet#5]
Unable to request funds to an account that hasn't been created yet.

Expected behavior would be to create an account if funds are requested to faucet.

account in question that was trying to fund already has funds:

https://explorer.movementlabs.xyz/account/0x17f3dee4a661d0e2b49728e10dae824bd45b055715946ffaa82471aa78cbb27e?network=mvmt_m1

Also attempting with other existent wallets leads to the same behavior.

Inexistent wallets also leads to the same behavior. Aptos Devnet is not behaving in the same way, using faucetClient leads to funding of accounts. Example of account creation and funding:

https://explorer.movementlabs.xyz/account/0xe59806d20415f2d40d1976f8ae9fea0614dd5978c98714434f8d42ec329f8992/?network=mvmt_m1

https://explorer.aptoslabs.com/account/0xe59806d20415f2d40d1976f8ae9fea0614dd5978c98714434f8d42ec329f8992?network=devnet

![image](https://github.com/movemntdev/faucet/assets/6299375/e1b638f1-2339-4a9d-b9d9-7204b793a04d)

https://faucet.movementlabs.xyz/?network=testnet

**Comment by danMove** [comment]
I found you in this web page using the faucet url to https://devnet.m1.movementlabs.xyz/v1/mint? & address = 0 xbf37798ec90ed4b98e146ee0250510debc69fa4a7a3c69811c503bb44c6a059f, if you remove &, I believe it can be successful. You can refer to the https://evm-bridge.devnet.m1.movementlabs.xyz/#/Faucet

**Comment by 0xPrimata** [comment]
gm @danMove thanks for the response!
The issue we are having is with the global signer, we were requesting faucet 

```
// NOTE: this is a private key for the faucet account, do not use it for anything else
const PRIVATE_KEY = "0xb6003d3fe766b8b98700a1f6ba71258043f9b9a39052631341ca5bd2e336473b";
const PUBLIC_KEY = "0xbf37798ec90ed4b98e146ee0250510debc69fa4a7a3c69811c503bb44c6a059f";
// const encoder = new TextEncoder(); // This is a built-in JavaScript API for encoding text

export const GLOBAL_SIGNER = AptosAccount.fromAptosAccountObject({
  privateKeyHex: PRIVATE_KEY,
  publicKeyHex: PUBLIC_KEY,
  address: "0x348116b94c9b734068cd07635c969fd724e5aa08fb63fd2ea52fd7d7e35b0fde"
});

export async function requestFaucetWithGlobalSigner(
  aptosClient: AptosClient,
  faucetClient: FaucetClient,
  coinClient: CoinClient,
  faucetUrl: string,
  address: string,
): Promise<any> {

  // double up the coins
  Promise.all([
    await requestFaucet(
      aptosClient,
      faucetClient,
      faucetUrl,
      PUBLIC_KEY,
    ),
    await requestFaucet(
      aptosClient,
      faucetClient,
      faucetUrl,
      PUBLIC_KEY,
    )
  ]);

  const tx = await coinClient.transfer(
    GLOBAL_SIGNER,
    address,
    1000000000
  );
  console.log(tx);
  return tx;

}
```

Error:

```
Error: {"message":"\"Account not found by Address(0x348116b94c9b734068cd07635c969fd724e5aa08fb63fd2ea52fd7d7e35b0fde) and Ledger version(1728124)\"","code":404,"error_code":"account_not_found","vm_error_code":null}
```

**Comment by l-monninger** [comment]
Should be resolved by #6
